toolkit/imageconfigs/packagelists/core-packages-container.json

```diff
@@ -1,5 +1,6 @@
 {
     "packages": [
-        "core-packages-container"
+        "core-packages-container",
+        "php"
```

Comment:
This looks fishy as it would add PHP to our base containers, which are expected to contain just a basic set of packages. I'd double-check if this is what we want and if we want it here instead of inside the `core-packages-container` package.

Reply:
Oops, I added this to git by accident. Meant to just keep it local for testing the PHP container build.